build/shtool

```diff
@@ -1583,9 +1583,9 @@ platform )
                            sed -e 's/^://' \
                                -e 's;\[\([^]]*\)\];\1;g' \
                                -e 's;<\([^>]*\)>;\1;g' \
-                               -e "s; ;��;g" \
```

Comment:
I don't know what characters this was referring to, but they shouldn't change. Try editing the file in an editor that preserves special characters.